lib/matplotlib/axes/_axes.py

```diff
@@ -1873,7 +1873,7 @@ def bar(self, x, height, width=0.8, bottom=None, *, align="center",
 
             - scalar: symmetric +/- values for all bars
             - shape(N,): symmetric +/- values for each bar
-            - shape(2,N): separate + and - values for each bar
+            - shape(2,N): First row lower errors, second row upper errors.
```

Comment:
Not a fan of the phrasing here, particularly given the line just before.  Also, the other entryes don't have a capital at the beginning.  

```
- shape(2,N): separate +/- values for each bar, the first row contains the - value.  
```